data-shaping/output/output.py

```diff
@@ -295,7 +295,7 @@ def transfer_exit_assessment_record(path, sheets):
             comments.append(row['OSCMNT'])
 
         ears = pd.DataFrame(
-            data={"id": ids, "training_record_id": training_ids, "date": dates, "type_id": types, "result_id": results, "comments": comments}
+            data={"id": ids, "training_record_id": training_ids, "date": dates, "type_id": types, "outcome_id": results, "comments": comments}
```

Comment:
for unfathomable to me reasons - using `result_id` makes OutSystems complain about circular FK dependency....